source/uwp/Renderer/AdaptiveCardRenderer.vcxproj

```diff
@@ -114,6 +114,7 @@
     </ClCompile>
   </ItemDefinitionGroup>
   <ItemGroup>
+    <ClInclude Include="..\..\shared\cpp\ObjectModel\MarkDownParser.h" />
```

Comment:
><ClInclude Include="..\..\shared\cpp\ObjectModel\MarkDownParser.h" /> [](start = 4, length = 69)

This project does not include any shared object model files. Please add them to the ObjectModelProjection project instead.

Reply:
Thanks

---
In reply to: [154217414](https://github.com/Microsoft/AdaptiveCards/pull/1003#discussion_r154217414) [](ancestors = 154217414)